readme.md

```diff
@@ -67,6 +67,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [IPFS](https://github.com/ipfs/awesome-ipfs)
 - [Fuse](https://github.com/vinkla/awesome-fuse)
 - [Heroku](https://github.com/ianstormtaylor/awesome-heroku)
+- [Raspberry Pi](https://github.com/thibmaek/awesome-raspberrypi)
```

Comment:
@sindresorhus all fixes required on the actual repo have been made, repo has also been renamed to awesome-raspberry-pi.
@lilorox I haven't been able to contribute to this PR itself, so if you'd be so kind to change what sindresorhus requests here while also updating the url that would be splendid 😄